src/viam_mlmodelservice_triton_impl.cpp

```diff
@@ -424,6 +448,10 @@ class Service : public vsdk::MLModelService, public vsdk::Stoppable, public vsdk
 
         const auto& attributes = state->configuration.attributes();
 
+        // If we're reconfiguring and have an old model name symlinked into our module workspace,
+        // remove it before setting up the new repo.
+        remove_symlink_mlmodel_(state);
```

Comment:
If we reconfigure but don't change models, I wonder if this will cause Triton to throw out the "old" model and reload the "new" model. Then again, reconfiguring should be rare, so it's not the end of the world if we waste some time in here.